scene/surface-placement/src/main/resources/surface_placement/style.css

```
@@ -1,7 +1,38 @@
+.panel-region {
+  -fx-background-color: rgba(0, 0, 0, 0.3);
+  -fx-padding: 10;
+}
+
 .panel-region .label {
   -fx-text-fill: white;
+  -fx-padding: 10;
+}
+
+.toggle-controls {
+  -fx-max-width: 210;
+  -fx-max-height: 120;
+  -fx-spacing: 6;
+}
+
+.toggle-controls .radio-button {
+  -fx-text-fill: white;
+}
+
+.slider-controls {
+  -fx-max-width: 90;
```

when viewed on a Mac, this width slightly truncates the `Z-Value:` label (even though it looks fine on Windows!). A value of 95 however corrects it. 
```suggestion
  -fx-max-width: 95;
```